test/stress-tests-kwok/gpu-tests.md

```diff
@@ -0,0 +1,160 @@
+## MCAD GPU Request Performance test with KWOK
+This experiment assumes that you have a KWOK controller as well as MCAD controller running inside a kind cluster. If not, follow [this](https://github.com/vishakha-ramani/multi-cluster-app-dispatcher/blob/main/test/perf-test/simulatingnodesandappwrappers.md) for installation instructions.
+The MCAD service for gpu requests is little weird . Here is the experiment I did and what I observed:
```

Comment:
`for gpu requests is little weird` can we change the wording and give the user a little more insight into the use case we ought to test, please?